patterns/featured-products-fresh-and-tasty.php

```diff
@@ -0,0 +1,130 @@
+			<!-- wp:column {"width":"40%","style":{"spacing":{"blockGap":"0"}},"layout":{"type":"constrained","justifyContent":"right"}} -->
+			<div class="wp-block-column" style="flex-basis:40%">
+				<!-- wp:paragraph {"align":"right","fontSize":"small"} -->
+				<p class="has-text-align-right has-small-font-size"><?php esc_html_e( 'from $1.99', 'woo-gutenberg-products-block' ); ?></p>
+				<!-- /wp:paragraph -->
+			</div>
+			<!-- /wp:column -->
+		</div>
+		<!-- /wp:columns -->
+	</div>
+	<!-- /wp:column -->
+
+	<!-- wp:column -->
+	<div class="wp-block-column">
+		<!-- wp:image {"sizeSlug":"full","linkDestination":"none"} -->
+		<figure class="wp-block-image size-full">
+			<img src="<?php echo esc_url( plugins_url( 'images/pattern-placeholders/fresh-organic-tomatoes.png', dirname( __FILE__ ) ) ); ?>" alt="<?php esc_attr_e( 'Placeholder image used to represent a product being showcased as Fresh Organic Tomatoes.', 'woo-gutenberg-products-block' ); ?>" />
+		</figure>
+		<!-- /wp:image -->
+
+		<!-- wp:columns {"isStackedOnMobile":false} -->
+		<div class="wp-block-columns is-not-stacked-on-mobile">
+			<!-- wp:column {"width":"60%","style":{"spacing":{"blockGap":"0"}},"layout":{"type":"constrained","justifyContent":"left"}} -->
+			<div class="wp-block-column" style="flex-basis:60%">
+				<!-- wp:paragraph {"fontSize":"small"} -->
+				<p class="has-small-font-size"><?php esc_html_e( 'Fresh Organic Tomatoes', 'woo-gutenberg-products-block' ); ?></p>
+				<!-- /wp:paragraph --></div>
+			<!-- /wp:column -->
+			<!-- wp:column {"width":"40%","style":{"spacing":{"blockGap":"0"}},"layout":{"type":"constrained","justifyContent":"right"}} -->
+			<div class="wp-block-column" style="flex-basis:40%">
+				<!-- wp:paragraph {"align":"right","fontSize":"small"} -->
+				<p class="has-text-align-right has-small-font-size"><?php esc_html_e( 'from $2.99', 'woo-gutenberg-products-block' ); ?></p>
+				<!-- /wp:paragraph -->
+			</div>
+			<!-- /wp:column -->
+		</div>
+		<!-- /wp:columns -->
+	</div>
+	<!-- /wp:column -->
+
+	<!-- wp:column -->
+	<div class="wp-block-column">
+		<!-- wp:image {"sizeSlug":"full","linkDestination":"none"} -->
+		<figure class="wp-block-image size-full">
+			<img src="<?php echo esc_url( plugins_url( 'images/pattern-placeholders/fresh-lettuce-washed.png', dirname( __FILE__ ) ) ); ?>" alt="<?php esc_attr_e( 'Placeholder image used to represent a product being showcased as Fresh Lettuce Washed.', 'woo-gutenberg-products-block' ); ?>" />
+		</figure>
+		<!-- /wp:image -->
+
+		<!-- wp:columns {"isStackedOnMobile":false} -->
+		<div class="wp-block-columns is-not-stacked-on-mobile">
+			<!-- wp:column {"width":"60%","style":{"spacing":{"blockGap":"0"}},"layout":{"type":"constrained","justifyContent":"left"}} -->
+			<div class="wp-block-column" style="flex-basis:60%">
+				<!-- wp:paragraph {"fontSize":"small"} -->
+				<p class="has-small-font-size"><?php esc_html_e( 'Fresh Lettuce (Washed)', 'woo-gutenberg-products-block' ); ?></p>
```

Comment:
Should we consider classes such as this one `has-small-font-size` as opinionated? IMO as long as we are not hardcoding font sizes (e.g. 12px) this should be a-ok, but curious to hear folks' opinions.